lochness/keyring/__init__.py

```diff
@@ -47,6 +47,36 @@ def box_api_token(Lochness, key):
             Lochness['keyring'][key]['API_TOKEN'])
 
 
+def rsync_token(Lochness, key):
+    '''get rsync token from keyring'''
```

Comment:
Is `rsync` an SSH mechanism? If it is, we only need private/public key pair to authenticate.

Reply:
Yes it is `ssh` mechanism. 

>Is rsync an SSH mechanism? If it is, we only need private/public key pair to authenticate.

Doesn't this require for the user to create the key pair?

Reply:
Yes, it does. Thinking again, writing the password in keyring might be easier. No action is necessary.